src/main/java/com/usharesoft/jenkins/steps/InstallStep.java

```diff
@@ -8,33 +10,85 @@
 import org.apache.commons.compress.compressors.gzip.GzipCompressorInputStream;
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.IOUtils;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
 
+import java.io.BufferedReader;
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
 import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.StringReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.NoRouteToHostException;
 import java.net.URL;
+import java.util.Base64;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
 
 import hudson.AbortException;
+import hudson.FilePath;
 import hudson.util.ArgumentListBuilder;
 
-public class InstallStep extends UForgeStep {
```

Comment:
Why not removing UForgeStep completely? HammrStep extends UForgeStep but I think it doesn't bring anything